base.py

```diff
@@ -18,12 +18,12 @@ def __init__(
         self.engineers: int = 0
         self.scientists: int = 0
 
-    def add_base_pop(self, pop: dict):
-        self.pioneers += pop["pioneers"]
-        self.settlers += pop["settlers"]
-        self.technicians += pop["technicians"]
-        self.engineers += pop["engineers"]
-        self.scientists += pop["scientists"]
+    def add_base_pop(self, pop: dict, amount: int):
```

Comment:
Nit: Use a different variable name than `pop` due to `pop` being a method for retrieving and removing data from certain data types.